# English translations for vigilo-correlator.
# Copyright (C) 2011 ORGANIZATION
# This file is distributed under the same license as the vigilo-correlator
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2011.
#
msgid ""
msgstr ""
"Project-Id-Version: vigilo-correlator 2.0.1\n"
"Report-Msgid-Bugs-To: contact@projet-vigilo.org\n"
"POT-Creation-Date: 2011-06-10 17:03+0200\n"
"PO-Revision-Date: 2011-06-10 17:03+0200\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: en <LL@li.org>\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 0.9.4\n"

#: src/twisted/plugins/vigilo_correlator.py:36
msgid "The topology has been reloaded."
msgstr "The topology has been reloaded."

#: src/twisted/plugins/vigilo_correlator.py:53
msgid ""
"Could not set signal handlers. The correlator may not be able to shutdown"
" cleanly"
msgstr ""
"Could not set signal handlers. The correlator may not be able to shutdown"
" cleanly"

#: src/vigilo/correlator/correvent.py:83
#, python-format
msgid ""
"Ignoring event #%(idevent)d on (%(host)r, %(service)r) as requested by "
"the correlation rules"
msgstr ""
"Ignoring event #%(idevent)d on (%(host)r, %(service)r) as requested by "
"the correlation rules"

#: src/vigilo/correlator/correvent.py:130
#, python-format
msgid "Updating existing correlated event (%r)"
msgstr "Updating existing correlated event (%r)"

#: src/vigilo/correlator/correvent.py:133
#, python-format
msgid "Got a reference to a non-existent correlated event (%r), adding as new"
msgstr "Got a reference to a non-existent correlated event (%r), adding as new"

#: src/vigilo/correlator/correvent.py:143
#, python-format
msgid "Raw event ignored. Reason: status = %r"
msgstr "Raw event ignored. Reason: status = %r"

#: src/vigilo/correlator/correvent.py:169
#, python-format
msgid ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"
msgstr ""
"Got a reference to a nonexistent correlated event (%r), skipping this "
"aggregate"

#: src/vigilo/correlator/correvent.py:212
msgid "Creating new correlated event"
msgstr "Creating new correlated event"

#: src/vigilo/correlator/correvent.py:328
msgid "Sending correlated event to syslog"
msgstr "Sending correlated event to syslog"

#: src/vigilo/correlator/db_insertion.py:49
#, python-format
msgid "Received request to add an event on HLS \"%s\""
msgstr "Received request to add an event on HLS \"%s\""

#: src/vigilo/correlator/db_insertion.py:57
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping event"
msgstr ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping event"

#: src/vigilo/correlator/db_insertion.py:89
#, python-format
msgid "Updating event %r"
msgstr "Updating event %r"

#: src/vigilo/correlator/db_insertion.py:95
#, python-format
msgid ""
"Ignoring request to create a new event with state \"%s\" (nothing "
"alarming here)"
msgstr ""
"Ignoring request to create a new event with state \"%s\" (nothing "
"alarming here)"

#: src/vigilo/correlator/db_insertion.py:102
msgid "Creating new event"
msgstr "Creating new event"

#: src/vigilo/correlator/db_insertion.py:105
msgid "Multiple matching events found, skipping."
msgstr "Multiple matching events found, skipping."

#: src/vigilo/correlator/db_insertion.py:150
#: src/vigilo/correlator/db_insertion.py:187
#: src/vigilo/correlator/db_insertion.py:231
#: src/vigilo/correlator/db_insertion.py:251
msgid "Got exception"
msgstr "Got exception"

#: src/vigilo/correlator/db_insertion.py:170
#, python-format
msgid "Got a reference to a non configured high-level service (%(service)r)"
msgstr "Got a reference to a non configured high-level service (%(service)r)"

#: src/vigilo/correlator/db_insertion.py:205
#, python-format
msgid ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping state"
msgstr ""
"Got a reference to a non configured item (%(host)r, %(service)r), "
"skipping state"

#: src/vigilo/correlator/db_insertion.py:268
#, python-format
msgid "Merging aggregate #%(src)d with aggregate #%(dest)d"
msgstr "Merging aggregate #%(src)d with aggregate #%(dest)d"

#: src/vigilo/correlator/db_insertion.py:279
msgid ""
"merge_aggregates: Got a reference to a nonexistent source aggregate, "
"aborting"
msgstr ""
"merge_aggregates: Got a reference to a nonexistent source aggregate, "
"aborting"

#: src/vigilo/correlator/db_insertion.py:289
#, python-format
msgid ""
"merge_aggregates: Got a reference to a nonexistent destination aggregate "
"%r, aborting"
msgstr ""
"merge_aggregates: Got a reference to a nonexistent destination aggregate "
"%r, aborting"

#: src/vigilo/correlator/handle_ticket.py:35
#, python-format
msgid ""
"handle_ticket: Trouble ticket message received. Timestamp = "
"%(timestamp)r. Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. "
"acknowledgement_status = %(ack_status)r. Message = %(message)r."
msgstr ""
"handle_ticket: Trouble ticket message received. Timestamp = "
"%(timestamp)r. Impacted HLS = %(hls)r. Ticket id = %(ticket_id)r. "
"acknowledgement_status = %(ack_status)r. Message = %(message)r."

#: src/vigilo/correlator/handle_ticket.py:57
#, python-format
msgid "handle_ticket: No matching trouble ticket found : %r"
msgstr "handle_ticket: No matching trouble ticket found : %r"

#: src/vigilo/correlator/handle_ticket.py:63
#, python-format
msgid "handle_ticket: Several events seem to be associated with this ticket : %r"
msgstr "handle_ticket: Several events seem to be associated with this ticket : %r"

#: src/vigilo/correlator/handle_ticket.py:66
#, python-format
msgid ""
"handle_ticket: The event %(event_id)r is associated with the given ticket"
" (%(ticket_id)r)"
msgstr ""
"handle_ticket: The event %(event_id)r is associated with the given ticket"
" (%(ticket_id)r)"

#: src/vigilo/correlator/handle_ticket.py:88
#, python-format
msgid "handle_ticket: Got exception while updating event %r history"
msgstr "handle_ticket: Got exception while updating event %r history"

#: src/vigilo/correlator/handle_ticket.py:92
#, python-format
msgid "handle_ticket: Event %r history updated successfully."
msgstr "handle_ticket: Event %r history updated successfully."

#: src/vigilo/correlator/memcached_connection.py:105
#, python-format
msgid "Deleted %(nb_deleted)d expired correlation contexts"
msgstr "Deleted %(nb_deleted)d expired correlation contexts"

#: src/vigilo/correlator/memcached_connection.py:148
#, python-format
msgid "Establishing connection to MemcacheD server (%s)..."
msgstr "Establishing connection to MemcacheD server (%s)..."

#: src/vigilo/correlator/memcached_connection.py:167
#, python-format
msgid "Trying to set value '%(value)s' for key '%(key)s'."
msgstr "Trying to set value '%(value)s' for key '%(key)s'."

#: src/vigilo/correlator/memcached_connection.py:217
#, python-format
msgid "Trying to get the value of the key '%(key)s'."
msgstr "Trying to get the value of the key '%(key)s'."

#: src/vigilo/correlator/memcached_connection.py:270
#, python-format
msgid "Trying to delete the key '%(key)s'."
msgstr "Trying to delete the key '%(key)s'."

#: src/vigilo/correlator/registry.py:58
#, python-format
msgid ""
"Rule %(name)s (%(class)s) has already been registered, ignoring attempt "
"to re-register it."
msgstr ""
"Rule %(name)s (%(class)s) has already been registered, ignoring attempt "
"to re-register it."

#: src/vigilo/correlator/registry.py:67
#, python-format
msgid "The rule %(depended)r must be loaded before %(dependent)r."
msgstr "The rule %(depended)r must be loaded before %(dependent)r."

#: src/vigilo/correlator/registry.py:75
#, python-format
msgid "Successfully registered rule %r"
msgstr "Successfully registered rule %r"

#: src/vigilo/correlator/registry.py:140
msgid ""
"Singleton has already been instanciated, ignoring attempt to create a new"
" instance."
msgstr ""
"Singleton has already been instanciated, ignoring attempt to create a new"
" instance."

#: src/vigilo/correlator/registry.py:175
#, python-format
msgid "Not a valid rule name \"%s\""
msgstr "Not a valid rule name \"%s\""

#: src/vigilo/correlator/registry.py:181
#, python-format
msgid "Unable to load rule \"%(name)s\" (%(class)s)"
msgstr "Unable to load rule \"%(name)s\" (%(class)s)"

#: src/vigilo/correlator/actors/executor.py:58
#, python-format
msgid "Rule %(rule_name)s timed out"
msgstr "Rule %(rule_name)s timed out"

#: src/vigilo/correlator/actors/rule_dispatcher.py:110
msgid ""
"No information from Vigiconf concerning the topology's last modification "
"date. Therefore the topology has NOT been rebuilt."
msgstr ""
"No information from Vigiconf concerning the topology's last modification "
"date. Therefore the topology has NOT been rebuilt."

#: src/vigilo/correlator/actors/rule_dispatcher.py:122
msgid "Topology has been reloaded."
msgstr "Topology has been reloaded."

#: src/vigilo/correlator/actors/rule_dispatcher.py:189
#, python-format
msgid "Skipping unrecognized item (%s)"
msgstr "Skipping unrecognized item (%s)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:212
msgid "Received invalid XMPP item ID (None)"
msgstr "Received invalid XMPP item ID (None)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:246
msgid "Error while retrieving the network's topology"
msgstr "Error while retrieving the network's topology"

#: src/vigilo/correlator/actors/rule_dispatcher.py:255
msgid "Error while adding an entry in the HLS history"
msgstr "Error while adding an entry in the HLS history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:261
msgid "Error while adding an entry in the history"
msgstr "Error while adding an entry in the history"

#: src/vigilo/correlator/actors/rule_dispatcher.py:268
msgid "Error while saving state"
msgstr "Error while saving state"

#: src/vigilo/correlator/actors/rule_dispatcher.py:316
msgid "Connection lost"
msgstr "Connection lost"

#: src/vigilo/correlator/actors/rule_dispatcher.py:362
msgid "Error while saving the correlated event"
msgstr "Error while saving the correlated event"

#: src/vigilo/correlator/actors/rule_dispatcher.py:392
#, python-format
msgid "Correlation failed for message #%(id)s (%(payload)s)"
msgstr "Correlation failed for message #%(id)s (%(payload)s)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:413
#, python-format
msgid "Unable to store correlated alert for message #%(id)s (%(payload)s)"
msgstr "Unable to store correlated alert for message #%(id)s (%(payload)s)"

#: src/vigilo/correlator/actors/rule_dispatcher.py:443
#, python-format
msgid "Executing %s with %r, %r"
msgstr "Executing %s with %r, %r"

#: src/vigilo/correlator/actors/rule_dispatcher.py:446
#, python-format
msgid "%s. The message will be handled once more."
msgstr "%s. The message will be handled once more."

#: src/vigilo/correlator/actors/rule_runner.py:82
#, python-format
msgid ""
"Got an exception while running rule %(rule)s. Running the correlator in "
"the foreground (%(prog)s -n) may help troubleshooting"
msgstr ""
"Got an exception while running rule %(rule)s. Running the correlator in "
"the foreground (%(prog)s -n) may help troubleshooting"

#: src/vigilo/correlator/amp/correlator.py:20
#, python-format
msgid "Sending this payload to the XMPP bus: %r"
msgstr "Sending this payload to the XMPP bus: %r"

#: src/vigilo/correlator/amp/correlator.py:26
#, python-format
msgid ""
"Registering post-correlation callback function \"%(fn)s\" for alert with "
"ID %(id)s"
msgstr ""
"Registering post-correlation callback function \"%(fn)s\" for alert with "
"ID %(id)s"

#: src/vigilo/correlator/rules/example.py:36
#, python-format
msgid "id %(id)s payload %(payload)s"
msgstr "id %(id)s payload %(payload)s"

#: src/vigilo/correlator/rules/svc_on_host_down.py:110
msgid "Host is down"
msgstr "Host is down"

#: src/vigilo/correlator/test/test_logging.py:271
msgid "Received 'WARNING' message on lls1"
msgstr "Received 'WARNING' message on lls1"

#: src/vigilo/correlator/test/test_logging.py:292
msgid "Received 'CRITICAL' message on lls1"
msgstr "Received 'CRITICAL' message on lls1"

